int8 bogie_connection_1
int8 bogie_connection_2
int8 bogie_connection_3